kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/FormServiceRestOnlyIntegrationTest.java

```diff
@@ -0,0 +1,251 @@
+import org.kie.server.api.model.KieContainerResource;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.api.model.instance.TaskSummary;
+import org.kie.server.api.model.instance.TaskSummaryList;
+import org.kie.server.api.model.type.JaxbLong;
+import org.kie.server.api.rest.RestURI;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.integrationtests.config.TestConfig;
+import org.kie.server.integrationtests.shared.RestOnlyBaseIntegrationTest;
+
+import static org.junit.Assert.*;
+import static org.kie.server.api.rest.RestURI.*;
+
+public class FormServiceRestOnlyIntegrationTest extends RestOnlyBaseIntegrationTest {
+
+    private static ReleaseId releaseId = new ReleaseId("org.kie.server.testing", "definition-project",
+            "1.0.0.Final");
+
+    private static final String CONTAINER_ID = "definition-project";
+
+    private static final String HIRING_PROCESS_ID = "hiring";
+
+    @ClassRule
+    public static ExternalResource StaticResource = new DBExternalResource();
+
+    @BeforeClass
+    public static void buildAndDeployArtifacts() {
+
+        buildAndDeployCommonMavenParent();
+        buildAndDeployMavenProject(ClassLoader.class.getResource("/kjars-sources/definition-project").getFile());
+
+    }
+
+    @Override
+    protected KieServicesClient createDefaultClient() {
+        KieServicesClient servicesClient = super.createDefaultClient();
+        return servicesClient;
+    }
+
+    @Before
+    public void cleanup() {
+        cleanupSingletonSessionId();
+
+    }
+
+    @Test
+    public void testGetProcessFormTest() throws Exception {
+        KieContainerResource resource = new KieContainerResource(CONTAINER_ID, releaseId);
+        assertSuccess(client.createContainer(CONTAINER_ID, resource));
+
+        Map<String, Object> valuesMap = new HashMap<String, Object>();
+        valuesMap.put(RestURI.CONTAINER_ID, resource.getContainerId());
+        valuesMap.put(RestURI.PROCESS_ID, HIRING_PROCESS_ID);
+
+        ClientResponse<?> response = null;
+        try {
+
+            ClientRequest clientRequest = newRequest(build(TestConfig.getKieServerHttpUrl(), FORM_URI + "/" + PROCESS_FORM_GET_URI, valuesMap))
+                    .header("Content-Type", getMediaType().toString())
+                    .header("Accept", getMediaType().toString());
+            logger.info( "[GET] " + clientRequest.getUri());
+
+            response = clientRequest.get();
+            Assert.assertEquals(Response.Status.OK.getStatusCode(), response.getStatus());
+
+            String result = response.getEntity(String.class);
+            logger.debug("Form content is '{}'", result);
+            assertNotNull(result);
+            assertFalse(result.isEmpty());
+
+        } catch (Exception e) {
+            throw new ClientResponseFailure(e, response);
+        } finally {
+            if (response != null) {
+                response.releaseConnection();
```

Comment:
Not directly related to this PR, but this clean-up could be moved to @After method to remove some redundancy.